### ISSUE TITLE
luarocks installation does not work by defualt.

### ISSUE THREAD

**Issue by StringManolo** [termux/termux-packages#6015]
Since the lua packages names are:
lua52
lua53
lua54
luajit

The shebang on the luarocks file should point by default to any of those elfs?

Playing around i set the shebang /data/data/com.termux/files/usr/bin/luarocks from ABSOLUTESHEBANG/bin/lua to #!/usr/bin/lua5.3 (my installed version using pkg install lua53) so i can run the command $ luarocks 

By default without change the shebang i had an error saying something about bash. Tested it on a fresh termux installation and on my usual one. 

Reproduce:
1. Install your favourite lua version using pkg. Ex:
pkg install lua54

2. Install luarocks to download any lua module.
pkg install luarocks

3. Try to download a module.
luarocks install luasocket

Bash error...

**Comment by fastcoding** [comment]
that's only an small issue, bcos the usr/bin/lua symbol link to lua5.3 is missing, which is referenced from usr/bin/luarocks script line#1.

I just found the big issue is that this luarocks package version seems having been modified/customized. it's hardcoded to be used only with lua5.3,  I have no way to make it install/build package for luajit like before.

in deps.lua  Line#534: 

```lua
function deps.check_lua(vars)
   vars.LUA_INCDIR = "/data/data/com.termux/files/usr/include/lua5.3"
   vars.LUALIB = "lua5.3"
   return true
end
```

**Comment by dauuricus** [comment]
the small issue  ...

@localhost ~> ln -s /data/data/com.termux/files/usr/bin/lua5.3 /data/data/com.termux/files/usr/bin/lua

**Comment by xtkoba** [comment]
I'm pretty much sure we can fix this by using `update-alternatives` (#4137).

Any objections?
